scripts/pkg_in_pipe/pkg_in_pipe.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+
+import os
+import sys
+from datetime import datetime
+
+import koji
+import requests
+
+def print_header(out):
+    print('''
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>XCP-ng Package Update</title>
+  <script src="https://cdn.tailwindcss.com"></script>
+</head>
+<body class="bg-gray-400 text-center">
+''', file=out)
+
+def print_footer(out):
+    now = datetime.now()
+    print(f'''
+Last generated at {now}. Generated every 5 minutes.
```

Comment:
While the "last generated" is useful info, hardcoding the interval of an external cron job does not feel right.

Reply:
do you think it should be a command line option?

Reply:
@nathanael-h as a consequence of this change, we'll have to add `--generated-info "Generated every 5 minutes."` on the command line when calling `pkg_in_pipe.py`

Reply:
When the delay is short and people see that it was last generated less than 5 minutes ago, I think passing a parameter to tell them "it's generated every 5 minutes" is a bit overkill, but now that it's done, why not :)